site-content/source/modules/ROOT/pages/download.adoc

```diff
@@ -124,6 +124,21 @@ https://www.apache.org/dyn/closer.lua/cassandra/3.0.30/apache-cassandra-3.0.30-b
 ==== The PGP keys used to sign releases are available in this https://downloads.apache.org/cassandra/KEYS[KEYS,window=_blank] file.
 ----------
 
+{empty} +
+
+[openblock, inline100 inline-top]
+----------
+
+[discrete]
+=== Cassandra Java Drivers
+
+Official Java drivers for Cassandra are distributed on Maven repositories.
+
+[.btn.btn--alt]
+https://mvnrepository.com/artifact/org.apache.cassandra/java-driver-core[Download from MVN Repository,window=blank]
```

Comment:
The ASF's policy about a project's releases are strict, and this doesn't adhere. 

A lot of it comes as a surprise to many, but t's important committers and the PMC are knowledgable to it.  It is the security and IP governance guarantees that ASF is renown for. https://www.apache.org/legal/release-policy.html

A few things in summary:
- the downloads page, and the project in general, can only announce our official releases to the public
- a release is the source tarball, that has been signed, checksummed and voted on
- the release is found in https://downloads.apache.org/cassandra (svn repository)
- the release artefact is linked to via https://www.apache.org/dyn/closer.lua/cassandra/
- the release signage and checksums are linked to directly via https://downloads.apache.org/cassandra/
- a link to this release artefact, and its signage and checksums must exist
- binary artefacts are referred to as convenience artefacts, built from the source release
- the key used to sign all artefacts must be rsa 4096 bit 
- the key's private must only exist on hardware the committer has physical possession and control of and exclusively full administrative/superuser access to
- the key's public must exist in the project's KEYS file and be available on public pgp servers

Reply:
fyi @absurdfarce

Reply:
Ok, so in summary, the driver download shouldn't be from a Maven repo like Maven Central?

Reply:
😁 
It can still list Maven Central.  Links to our convenience binaries, where ever they may be found, is valuable (and really what users are looking for).  See if our downloads page (and all other apache project download pages) make a bit more sense now…

Reply:
@absurdfarce @michaelsembwever I spoke with Bret about which URL to use, and have updated the PR incorporating both of your inputs here.  Let me know if this is closer.